domain-management/src/main/java/org/jboss/as/domain/management/audit/SyslogAuditLogHandlerResourceDefinition.java

```diff
@@ -412,7 +412,7 @@ protected void revertUpdateToRuntime(OperationContext context, ModelNode operati
                 addr = addr.subAddress(0, addr.size() - 1);
                 auditLogger.updateSyslogHandlerReconnectTimeout(Util.getNameFromAddress(addr), valueToRestore.asInt());
             } else if (attributeName.equals(KEYSTORE_PASSWORD_CREDENTIAL_REFERENCE_NAME) || attributeName.equals(KEY_PASSWORD_CREDENTIAL_REFERENCE_NAME)) {
-                rollbackCredentialStoreUpdate(getAttributeDefinition(attributeName), context, valueToRevert);
+                rollbackCredentialStoreUpdate(context.getResourceRegistration().getAttributeAccess(PathAddress.EMPTY_ADDRESS, attributeName).getAttributeDefinition(), context, valueToRevert);
```

Comment:
+1 The usage of this method is tricky one since it is still deprecated but not removed with this PR because it needs chances in WildFly too, however, we have to prevent that nobody uses it if we are removing the constructor with the attributes, that's my understanding.